src/main/java/frc/robot/commands/ShooterCommand.java

```diff
@@ -69,8 +69,9 @@ public void execute() {
         shooterSpeedController.setLaunchSpeed(ShooterMath
                 .determineLaunchSpeed(distance + INNER_DISTANCE_FROM_TARGET - CAMERA_DISTANCE_FROM_LAUNCHER));
 
+        double offset = limelight.getHorizontalTargetOffset();
         // Make sure the shooter is at speed before loading a power cell
-        if (shooterSpeedController.isAtSpeed()) {
+        if (shooterSpeedController.isAtSpeed() && Math.abs(offset) > TOP_GOAL_DEADBAND) {
```

Comment:
This looks to me like we'll intake/index a power cell when we're out of the deadband.  Wouldn't we want to do this while we're <= to the deadband?